### PR TITLE
VIEW: Delete post

### PR DESCRIPTION
**Pull request by rcruzfreelance**

### CODE REVIEW

**Comment by rcruzfreelance** [comment]
Is possible the conflicts has to do with the View folders I created in the project

**Comment by hassanhabib** [comment]
@rcruzfreelance you need to fix the merge conflict

**Comment by rcruzfreelance** [comment]
@hassanhabib merge conflict resolved

File: Taarafo.Portal.Web/Services/Views/PostViews/PostViewService.Exceptions.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            }
+            catch(Exception exception)
+            {
+                var failedPostViewServiceException =
+                    new FailedPostViewServiceException(exception);
+
+                throw CreateAndLogServiceException(failedPostViewServiceException);
+            }
+        }
+
```
Comment (hassanhabib) [inline Taarafo.Portal.Web/Services/Views/PostViews/PostViewService.Exceptions.cs:78]: Remove extra line

**Pull request merged**